django_shop/settings.py

```diff
@@ -10,20 +10,28 @@
 https://docs.djangoproject.com/en/1.8/ref/settings/
 """
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
+import ConfigParser
 
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+config = ConfigParser.ConfigParser()
+config.read(os.path.join(app.root_path, '../settings.ini')
 
+DATABASE_USER = config.get('database', 'DATABASE_USER')
+DATABASE_PASSWORD = config.get('database', 'DATABASE_PASSWORD')
+DATABASE_ENGINE = config.get('database', 'DATABASE_ENGINE')
+DATABASE_NAME = config.get('database', 'DATABASE_NAME')
```

Comment:
We must put this in the DATABASES dict. See https://docs.djangoproject.com/en/1.8/ref/settings/#databases